Helper/Vault.php

```diff
@@ -393,6 +394,10 @@ private function getVaultPaymentToken(
 
         if (isset($recurringModel)) {
             $details[self::TOKEN_TYPE] = $recurringModel;
+            // Set token cardHolderName for new Visa compliance requirements
+            if ($additionalData[self::CARDHOLDER_NAME] !== null) {
```

Comment:
Hey, just a suggestion, I think !empty() is more effective than !== null, since it checks for both empty and null.